hack/install_kustomize.sh

```diff
@@ -165,7 +165,7 @@ curl -sLO "$RELEASE_URL"
 
 tar xzf ./kustomize_v*_${opsys}_${arch}.tar.gz
 
-cp ./kustomize "$where"
+cp ./output/kustomize "$where"
```

Comment:
Since this script should be able to install any version of Kustomize (see line 32), I'd suggest something like this for backward compatibility:

```suggestion
if [[ -f ./output/kustomize ]]; then
  cp ./output/kustomize "$where"
elif [[ -f ./kustomize "$where" ]]; then
  cp ./kustomize "$where"
else
  echo "kustomize binary not found in the archive"
  exit 1
fi
```